extract/smil.es6

```diff
@@ -36,5 +36,7 @@ export function parseAll(items, manifest, metadata) {
 export default function smil(uri, manifest, metadata) {
   const items = getSmilFromManifest(manifest);
 
-  return fetchAll(uri, items, manifest).then(parseAll(items, manifest, metadata));
+  return fetchAll(uri, items, manifest)
+    .then(values => parseAll(items, manifest, metadata)(values))
+    .catch(error => console.error(error));
```

Comment:
We may want to do something with those errors later on. Let's wait until we use this and decide, shall we?